Lecture-14/src/main/java/GitUser.java

```diff
@@ -0,0 +1,103 @@
+public class GitUser {
+    private String login;
+    private int id;
+    private String node_id;
+    private String avatar_url;
+    private String gravatar_id;
```

Comment:
Do you really need all those fields? 
Use `@JsonIgnoreProperties(ignoreUnknown = true)`

Reply:
done